tool/common_mark_stats.json

```diff
@@ -425,7 +425,7 @@
   "474": "fail",
   "475": "strict",
   "476": "strict",
-  "477": "strict",
+  "477": "fail",
```

Comment:
Oh no! Why did this start failing?

Reply:
Good question!

Reply:
It only "started" failing the sense that this PR no longer moves it to "strict". [477](http://spec.commonmark.org/0.28/#example-477) is a weird test showing how quotes are to be parsed in certain cases. Since escape logic is still far from complete, fixing 477 is just out of scope for this PR. It "started" failing in my last push because of reworking escape logic (mostly for the better, but no longer fixing 477).